runelite-client/src/main/java/net/runelite/client/plugins/entityhider/EntityHiderConfig.java

```diff
@@ -202,8 +202,8 @@ default boolean hideThralls()
 	@ConfigItem(
 		position = 16,
 		keyName = "hideRandomEvents",
-		name = "Hide Random Events",
-		description = "Configures whether or not Random events are hidden"
+		name = "Hide Others Players' Random Events",
+		description = "Configures whether or not other player random events are hidden"
```

Comment:
This should probably be `description = "Configures whether or not other players' random events are hidden"`

Reply:
I thought so, too. I just copied the description format from "Hide Other Players' Pets"